### PR TITLE
Search by amenity/facility, list amenity facility

### PR DESCRIPTION
**Pull request by luqmaan**
- Show popup when clicking on feature list item
- Refactor feature list into ParkFeatureListItem
- Add feature list
- Hook up amenity and facility drop downs
- Add amenity/facility lookup

### CODE REVIEW

File: client/scss/main.scss [hunk 1]
```diff
@@ -2,6 +2,26 @@
...
+}
+
+.container {
+}
+
+.address {
+    text-align: right;
+}
+
+.name {
```
Comment (aaronmgdr) [inline client/scss/main.scss:24]: may i suggest 'park-name' as a className instead? its descriptive and less likely to cause css conflicts in the future.

Comment (luqmaan) [inline client/scss/main.scss:24]: Sure, or maybe `.park .name`?

**Comment by luqmaan** [comment]
@fremn I've merged your changes into here.

File: client/scss/main.scss (hunk 1 above)
Comment (aaronmgdr) [inline client/scss/main.scss:24]: there's a good reason to use  `park-name` instead.

http://markdotto.com/2012/02/16/scope-css-classes-with-prefixes/

basically while `.park .name` ensures that we don't effect `.name`  Outside of `.park`  it doesn't prevent conflicts Inside of `.park` eg 
main.css

```
.park .name  {font-weight: bold  }
.something-else .name {color: red }
```

html structure with just classnames

```
.park
     .name
     .something-else
         .name // element will be styled bold and red
```

Comment (aaronmgdr) [inline client/scss/main.scss:24]: honestly though im ok either way

**Comment by aaronmgdr** [comment]
:deciduous_tree:

File: client/scss/main.scss (hunk 1 above)
Comment (luqmaan) [inline client/scss/main.scss:24]: I'll use `.park-name`

**Pull request merged**